docs/release-notes.md

```diff
@@ -12,6 +12,68 @@ Welcome to Semgrep release notes. This document provides an overview of the chan
 
 ## March 2022
 
+### Version 0.85.0
+
+#### Additions
+
+##### C# improvement
+
+Semgrep uses the latest tree-sitter-c-sharp with support for most C# 10.0 features.
+
+##### HTML improvement
+
+Support for metavariables on tags (for example: `\&lt;$TAG\&gt;...\&lt;/$TAG\&gt;`). ([#4078](https://github.com/returntocorp/semgrep/issues/4078))
+
+##### Scala improvement
+
+The data-flow engine now handles expression blocks. Previously, Semgrep did not report some false negatives when run with taint analysis on expression blocks, which are now reported.
+
+##### Dockerfile improvement
+
+Allow for example `CMD …` to match both `CMD ls` and `CMD [&quot;ls&quot;]`. ([#4770](https://github.com/returntocorp/semgrep/issues/4770))
+
+##### Semgrep informs about used rules for multiple languages
+
+When scanning multiple languages, Semgrep now prints a table of how many rules and files are used for each language.
+
+#### Changes
+
+##### File targeting logic
+
+The following inconsistencies were fixed: ([#4776](https://github.com/returntocorp/semgrep/pull/4776))
+
+##### Explicitly targeted files are now affected by global filters
+
+Previously, explicitly targeted files (files that are directly passed to the command line) were unaffected by most global filters: global include or exclude patterns, and the file size limit. The `.semgrepignore` patterns do not affect explicitly targeted files. The described issue has been fixed.
```

Comment:
"The `.semgrepignore` patterns do not affect explicitly targeted files" -- is this an exception to the previous sentence?

In general, global exclude/include filters are now applied, except for .semgrepignore patterns?

Reply:
@s-santillan good question. It was hard to understand the original sentence as well. It ended up with a comma... maybe there should have been something more to say.

![Screenshot 2022-03-23 at 18 15 13](https://user-images.githubusercontent.com/26669307/159757520-cbf3aace-4ca6-4f8f-bfaa-d27a377eb3ed.jpg)

Reply:
The above sentence sounds better! Now I know that semgrepignore will not affect explicitly targeted files. Can we use the above sentence instead?

Reply:
@s-santillan this is from the changelog. 

Changing the sentence:
The `.semgrepignore` patterns do not affect explicitly targeted files.

To:
Now, the `semgrepignore.` patterns do not affect explicitly targeted files as well.

I hope that I understand what you mean. If not, suggest an edit directly.